node/src/daser.rs

```diff
@@ -620,8 +620,8 @@ fn random_indexes(square_width: u16, max_samples_needed: usize) -> HashSet<(u16,
     let mut rng = rand::thread_rng();
 
     while indexes.len() < max_samples_needed {
-        let row = rng.gen::<u16>() % square_width;
-        let col = rng.gen::<u16>() % square_width;
+        let row = rng.r#gen::<u16>() % square_width;
+        let col = rng.r#gen::<u16>() % square_width;
```

Comment:
rand v0.9 deprecates `gen()` and introduces `random()`. However tendermint is using rand v0.8, so I couldn't upgrade rand for the whole workspace.

Should I upgrade to rand v0.9 for the workspace and leave rand v0.8 in celestia-types?